docs/pages/get-started/start-developing.mdx

```diff
@@ -86,10 +86,14 @@ Expo Go is configured by default to automatically reload the app whenever a file
 
 ## File structure
 
-Below, you can get familiar with the default project's file structure:
+Below, you can get familiar with the project's default file structure. This is one of many possible ways to structure your project. The only enforced convention is **app** directory for routes with Expo Router. Feel free to rename other directories and files to structure your project however you like.
```

Comment:
Our goal on this doc is to get someone up to speed enough to make a change locally on a project, and this addition doesn't seem to help with that. I'm thinking as someone who doesn't know what Expo Router is, nor are they familiar with us. I suspect they just want to see if they can get something running.

Perhaps we could add a note in the **app** details saying it's a special-cased directory name, specifically for app navigation. That seems helpful for first-time developers to me. It also implicitly hints that the other directories are not special-cased.